fk/main.go

```diff
@@ -205,6 +238,10 @@ func ensureSchedulerStateMatchesConfig() {
 	}
 }
 
+func launchdFilename() string {
```

Comment:
it's a very bad smell that `fk/main.go` knows anything about the internal implementation details of the launchd scheduler!